packages/openstef-models/src/openstef_models/presets/forecasting_workflow.py

```diff
@@ -100,9 +100,9 @@ class ForecastingWorkflowConfig(BaseConfig):  # PredictionJob
     model_id: ModelIdentifier = Field(description="Unique identifier for the forecasting model.")
 
     # Model configuration
-    model: Literal["xgboost", "gblinear", "flatliner", "hybrid", "lgbm", "lgbmlinear"] = Field(
-        description="Type of forecasting model to use."
-    )  # TODO(#652): Implement median forecaster
+    model: Literal[
+        "xgboost", "gblinear", "flatliner", "stacking", "residual", "learned_weights", "lgbm", "lgbmlinear"
+    ] = Field(description="Type of forecasting model to use.")  # TODO(#652): Implement median forecaster
```

Comment:
We already have a preset for metamodels in openstef-meta package. Would it make sense to keep them separate so we don't have to sink two presets on changes to one ensemble model?